src/App/Pages/Accounts/HomePage.xaml

```diff
@@ -11,12 +11,16 @@
     <ContentPage.BindingContext>
         <pages:HomeViewModel />
     </ContentPage.BindingContext>
+    
+    <ContentPage.ToolbarItems>
+        <ToolbarItem Text="{u:I18n Close}" Clicked="Close_Clicked" Order="Primary" Priority="-1" />
+    </ContentPage.ToolbarItems>
```

Comment:
The close option (only visible when home is shown from the extension) is to prevent the user from becoming trapped in the popup should they not want to login for [reasons]...

![Screen Shot 2020-09-03 at 12 15 23 PM](https://user-images.githubusercontent.com/59324545/92141059-eee9a180-eddf-11ea-9dec-6d762f9ba899.png)
![Screen Shot 2020-09-03 at 12 14 40 PM](https://user-images.githubusercontent.com/59324545/92141074-f4df8280-eddf-11ea-99ff-b6263288e039.png)